altair/examples/grouped_bar_chart_with_error_bars.py

```diff
@@ -0,0 +1,27 @@
+"""
+Grouped Bar Chart with Error Bars
+---------------------------
```

Comment:
This underline needs to be the same length as the line above or the RST->HTML conversion will complain.